packages/query/src/bitcoin/balance/btc-native-segwit-balance.hooks.ts

```diff
@@ -0,0 +1,10 @@
+import { useMemo } from 'react';
+
+import { createBitcoinCryptoCurrencyAssetTypeWrapper } from '../address/address.utils';
+import { useGetBitcoinBalanceByAddress } from './btc-balance.hooks';
+
+// Balance is derived from a single query in address reuse mode
+export function useNativeSegwitBalance(address: string) {
+  const balance = useGetBitcoinBalanceByAddress(address);
+  return useMemo(() => createBitcoinCryptoCurrencyAssetTypeWrapper(balance), [balance]);
+}
```

Comment:
I think a hard rule for this package should be **never exporting wrapped queries**. 

This has caused problems within the extension. This should be a query with a `select`, so we can still access the internals of the query.